MathLib/LinAlg/Eigen/EigenLinearSolver.cpp

```diff
@@ -12,6 +12,8 @@
 #include <logog/include/logog.hpp>
 
 #ifdef USE_EIGEN_UNSUPPORTED
+#include <iostream> // to fix compiling errors in Eigen
```

Comment:
Please add a version of Eigen where this is needed in the comment. Could help later to eliminate unnecessary includes.